Microsoft.Toolkit.Uwp.UI.Controls/RadialGauge/RadialGauge.cs

```diff
@@ -469,6 +469,10 @@ protected override void OnValueChanged(double oldValue, double newValue)
         {
             OnValueChanged(this);
             base.OnValueChanged(oldValue, newValue);
+            if (FrameworkElementAutomationPeer.CreatePeerForElement(this) is RadialGaugeAutomationPeer peer)
+            {
+                peer.RaiseValueChangedEvent(oldValue, newValue);
+            }
```

Comment:
You shouldn't always create the automation peer here. You can let the runtime provide it for you.
The event should be sent only if someone is listening to the event (ie the narrator/screen reader is on). Unfortunately, we cannot check for all events so but using the most generic `LiveRegionChanged` is working.


You can use something as:

```csharp
if (AutomationPeer.ListenerExists(AutomationEvents.LiveRegionChanged))
 {
     var peer = FrameworkElementAutomationPeer.FromElement(strongElement) as RadialGaugeAutomationPeer;
    peer?.RaiseValueChangedEvent(oldValue, newValue);
}
```

Reply:
Good idea, updated the code now!